### ISSUE TITLE
Bump version after changing oauth2 dependency

### ISSUE THREAD

**Issue by aclemmensen** [ueberauth/ueberauth_github#52]
This commit f00372eb92c7d9ce2fcf7eb9b98fce9a03d1ca0a did not entail a version bump and a new package on Hex. Seems like that would have been nice - at least the Poison/Jason discrepancy caught me by surprise at runtime.

**Comment by yordis** [comment]
@aclemmensen if I am not mistaken, there is no release for this, so I am not sure what your issue is at the moment since I am having trouble why this affect you at the moment.

**Comment by aclemmensen** [comment]
Sorry, I was unclear. I installed the version of the strategy that is published to Hex, but I got a runtime error when parsing claims from OAuth2 saying Poison was unavailable. I didn't understand why I'd need Poison since the ueberauth docs and code says Jason is the default (and Jason was in fact installed).

Looking here I see that the commit mentioned above fixes this problem by upgrading the OAuth2 dependency and correctly passing along the chosen serializer. That's perfect as it fixes the problem I was encountering.

So basically I was just requesting that this very nice fix would be published to Hex.pm as well so I wouldn't need to depend on this github repo to get a working version of the code.

Sorry for the confusion. I hope the above makes my intent clear.
